central/complianceoperator/v2/scanconfigurations/service/service_impl_test.go

```diff
@@ -656,6 +661,190 @@ func (s *ComplianceScanConfigServiceTestSuite) TestRunReport() {
 	s.Equal(v2.ComplianceRunReportResponse_SUBMITTED, resp.RunState, "Failed to submit report")
 }
 
+func (s *ComplianceScanConfigServiceTestSuite) TestGetReportHistory() {
+	s.T().Setenv(features.ComplianceReporting.EnvVar(), "true")
+	if !features.ComplianceReporting.Enabled() {
+		s.T().Skipf("Skip test when the compliance reporting feature flag is disabled")
```

Comment:
Style: I think, that should be sufficient
```suggestion
		s.T().Skipf("compliance reporting feature flag is disabled")
```

Reply:
Done